src/safeposix/dispatcher.rs

```diff
@@ -185,12 +192,6 @@ pub extern "C" fn dispatcher(cageid: u64, callnum: i32, arg1: Arg, arg2: Arg, ar
         FCNTL_SYSCALL => {
             cage.fcntl_syscall(unsafe{arg1.dispatch_int}, unsafe{arg2.dispatch_int}, unsafe{arg3.dispatch_int})
         }
-        DUP_SYSCALL => {
-            cage.dup_syscall(unsafe{arg1.dispatch_int}, None)
-        }
-        DUP2_SYSCALL => {
-            cage.dup2_syscall(unsafe{arg1.dispatch_int}, unsafe{arg2.dispatch_int})
-        }
```

Comment:
Why'd you remove this?

Reply:
These are already there earlier in the dispatcher. Not sure why they are in there twice so I deleted them. If you look above these two a little bit, the exact same thing is there. Is there supposed to be two sets of dup's?

Reply:
Nope, you're good